README.md

```diff
@@ -30,56 +22,96 @@ Alternatively you could create it on [ElephantSQL](https://www.elephantsql.com/)
 2.  Create a new Heroku app using this Heroku buildpack:
 
     ```bash
-    heroku apps:create ${YOUR_APP_NAME} --buildpack https://github.com/PostgREST/postgrest-heroku
+    mkdir ${YOUR_APP_NAME}
+    cd ${YOUR_APP_NAME}
+    git init .
+
+    heroku apps:create ${YOUR_APP_NAME} --buildpack https://github.com/abernicchia-heroku/postgrest-heroku.git
```

Comment:
> https://github.com/abernicchia-heroku/postgrest-heroku.git
 
Oops, the above URL should be changed to https://github.com/PostgREST/postgrest-heroku.git right?

Reply:
yes, exactly!